lib/Dialect/SV/Transforms/PrettifyVerilog.cpp

```diff
@@ -256,6 +473,18 @@ void PrettifyVerilogPass::processPostOrder(Block &body) {
           processPostOrder(regionBlock);
     }
 
+    // Simplify assignments involving structures and arrays.
+    if (auto assign = dyn_cast<sv::PAssignOp>(op)) {
```

Comment:
Can we handle `BPAssignOp` too?

Reply:
We're not really using them at the moment - I'd like to enable that at a later point.